pkg/cortex/cortex.go

```diff
@@ -173,6 +174,10 @@ func (c *Config) Validate(log log.Logger) error {
 		return err
 	}
 
+	if c.HTTPPrefix != "" && !strings.HasPrefix(c.HTTPPrefix, "/") {
+		return errors.New("http_prefix should be empty or start with /")
```

Comment:
Define this error in a global variable `errInvalidHttpPrefix` and then use it to check the exact error in the unit test.

Reply:
Done! Yeah makes a lot of sense :)